firmware/application/ui_record_view.cpp

```diff
@@ -205,6 +205,11 @@ void RecordView::start() {
         return;
     }
 
+    // check for geo data, if present append filename with _GEO
+    if (latitude != 0 && longitude != 0 && latitude < 200 && longitude < 200) {
+        base_path.append_filename(u"_GEO");
```

Comment:
This is a really nice improvement.
I don't have a gps module to test it, just one small question:
would this append way break the filename number counting of next_filename_matching_pattern?
If not, then all look good to me.

Reply:
Based on my tests, yes it is ok!